### PR TITLE
Pass `cwd` option to dir-glob (#63)

### LINKED ISSUE #63: Directory expanding doesn't work with `cwd` option
```js
globby('components', { cwd: 'fixtures' })
// The code above doesn't work for this case:
// .
// └── fixtures/
//     └── components/
//         └── file
```

### PR DESCRIPTION
**Pull request by termosa**
Fixes #63: Directory expanding doesn't work with `cwd` option.

This commit doesn't break tests, but it doesn't do any changes unless [kevva/dir-glob#6](https://github.com/kevva/dir-glob/pull/6) won't be merged. I'll ping you, @sindresorhus, once it makes sense to release it.

### CODE REVIEW

**Comment by termosa** [comment]
Hi @sindresorhus !

`dir-glob` is updated to [version 2.2.0](https://github.com/kevva/dir-glob/commits/master) and now it supports `cwd` option.

Would you update `globby` please?
Thank you!

**Comment by kevva** [comment]
Could you add tests?

File: index.js [hunk 1]
```diff
@@ -53,15 +53,19 @@ const generateGlobTasks = (patterns, taskOpts) => {
 };
 
 const globDirs = (task, fn) => {
-	if (Array.isArray(task.opts.expandDirectories)) {
-		return fn(task.pattern, {files: task.opts.expandDirectories});
+	const opts = {};
```
Comment (kevva) [inline index.js]: You could write this function as:

```js
let opts = {cwd: task.opts.cwd};

if (Array.isArray(task.opts.expandDirectories)) {
	opts = Object.assign(opts, {files: task.opts.expandDirectories});
} else if (typeof task.opts.expandDirectories === 'object') {
	opts = Object.assign(opts, task.opts.expandDirectories);
}

return fn(task.pattern, opts);
```

**Comment by sindresorhus** [comment]
> Would you update globby please?

You can just bump it in this PR.

File: index.js (hunk 1 above)
Comment (termosa) [reply index.js]: `let opts = {cwd: task.opts.cwd};` is clear for me, but I don't understand why would you reassign `opts` variable if `Object.assign` is doing it. Are there any problems with it on some node versions?

Any way, I did what you asked and I replaced `'tmp'`literal with `tmp` variable. Hope you are agree on defining it as a constant.

**Comment by termosa** [comment]
Done. I also replaced `'tmp'` literal with `tmp` variable. Hope you are agree on defining it as a constant.

**Comment by kevva** [comment]
Sweet, looks good :).

**Pull request merged**